frontend/src/pages/hardwareProfiles/utils.ts

```diff
@@ -1,4 +1,224 @@
 import { HardwareProfileKind } from '~/k8sTypes';
+import { Identifier, IdentifierResourceType } from '~/types';
+import { determineUnit, splitValueUnit } from '~/utilities/valueUnits';
+import { hasCPUandMemory } from './manage/ManageNodeResourceSection';
+
+export type WarningNotification = {
+  warningStatus: boolean;
```

Comment:
I don't see a value adding a `warningStatus` prop here. You are setting it to `true` whenever there is a `parentWarningMessage`. Why not use that as the condition?